pkgs/os-specific/linux/cshatag/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildGoPackage, fetchFromGitHub, attr }:
+
+buildGoPackage rec {
+  pname = "cshatag";
+  version = "2019-12-03";
+
+  goPackagePath = "github.com/rfjakob/cshatag";
+  goDeps = ./deps.nix;
+
+  src = fetchFromGitHub {
+    owner = "rfjakob";
+    repo = pname;
+    rev = "b169f0a9dd35a7381774eb176d4badf64d403560";
+    sha256 = "16kam3w75avh8khkk6jfdnxwggz2pw6ccv6v7d064j0fbb9y8x0v";
+  };
+
+  nativeBuildInputs = [ attr ];
+
+  makeFlags = [ "PREFIX=$(out)" "GITVERSION=${version}" ];
+
+  postInstall = ''
+    cd go/src/${goPackagePath}
+    make install $makeFlags
+  '';
+
+  meta = with lib; {
+    description = "A tool to detect silent data corruption";
+    homepage = "https://github.com/rfjakob/cshatag";
+    license = licenses.mit;
+    platform = platforms.linux;
+  };
+
+}
+
```

```suggestion
}
```

GitHub does not show the final new line does it? If it does I did not mean to remove it.

---

Removed, thank you.